components/mqtt/server.go

```diff
@@ -471,7 +468,16 @@ func (s *Server) fetchAndPublishOutput(ctx context.Context, outputID iotago.Outp
 	if err != nil {
 		return
 	}
-	s.PublishOutput(ctx, resp.GetLedgerIndex(), resp.GetOutput(), false)
+	s.PublishOutput(ctx, iotago.SlotIndex(resp.GetOutput().GetSlotBooked()), resp.GetOutput(), false)
```

Comment:
I think the "slot booked" is the wrong index. It should be the commitment ID of the ledger instead IMO.

Reply:
This index is not used in the method, I'll remove it.